recipe/run_test.py

```diff
@@ -0,0 +1,26 @@
+import platform
+import subprocess
+
+import fastapi
+from fastapi import FastAPI, dependencies, middleware, openapi, security
+
+subprocess.run(["pip", "check"])
+
+app = FastAPI()
+
+
+@app.get("/")
+async def read_main():
+    return {"msg": "Hello World"}
+
+
+# Skip tests on s390x because of missing httpx package
+if platform.machine() != "s390x":
```

Comment:
as httpx itself supports s390x and it is absent because we skipped its build somehow, I think we should not do this in the test itself, and just skip it in the recipe by explicitly defining the test command with the comment to skip s390x

Reply:
The issue (or bug) is that conda-build will ignore both test/commands and run_test.py if they both exist. I would skip s390x at all because it's not required by the [JIRA ticket](https://anaconda.atlassian.net/browse/PKG-2612)

Reply:
I removed the condition `if platform.machine() != "s390x"` and skipped `s390x` in the `meta.yaml`